.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+^^ Provide a general summary of the issue in the title above. ^^ 
+
+**Description**
```

Comment:
Formatting comment: when I view this file, the new lines are not preserved, which makes this less readable. Presumably most people will replace the contents of each section anyway, but in the case that someone doesn't, it will look a bit cluttered.